### PR TITLE
[API-1402] Add support for batch create and delete

### PR DESCRIPTION
**Pull request by janvennemann**
JIRA: https://jira.appcelerator.org/browse/API-1402

### CODE REVIEW

File: lib/methods/createMany.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * @param {Arrow.Model} Model The model class being updated.
+ * @param {Array<Object>} values Array of attributes to set on the new models.
+ * @param {Function} callback Callback passed an Error object (or null if successful), and info about the batch operation.
+ * @throws {Error}
+ */
+exports.createMany = function (Model, values, callback) {
+	// Use batch delete if possible
+	var currentUser = this.user || this.baseUser;
+	var isAdminUser = currentUser.admin === 'true';
+	var modelSupportsBatchCreate = typeof Model.createMany === 'function';
```
Comment (vladimir-trifonov) [inline lib/methods/createMany.js]: By some reason typeof Model.createMany is always undefined on my side, even for the "custom_object" model and also during the 'should create models via batch operation' unit test. Some idea why it is like that?

Comment (janvennemann) [inline lib/methods/createMany.js]: Sorry, my bad. I missed the changes that are required to the arrowdb node sdk. You need to test this together with https://github.com/appcelerator/arrowdb-node-sdk/pull/12

**Comment by vladimir-trifonov** [comment]
@janvennemann  Should I merge it or you will?

**Comment by janvennemann** [comment]
@vladimir-trifonov, sure go a head! Just remember to merge https://github.com/appcelerator/arrowdb-node-sdk/pull/12 before this one ;)

**Comment by vladimir-trifonov** [comment]
@janvennemann You should merge it, because I can't merge arrowdb-node-sdk, I don't have enough rights.

**Pull request merged**